src/store/index.js

```diff
@@ -38,15 +40,18 @@ export const mutations = {
     state.enabledElements.push(enabledElement);
 
     if (brush) {
-      brush.mutations.SET_ELEMENT_VISIBLE(enabledElement);
+      brush.setters.setElementVisible(enabledElement);
     }
   }
 };
 
+export const modules = {
```

Comment:
For 3rd party support, perhaps we could do with an api hook that adds a module to `modules`. The 3rd party plugin would then be able to change its own state, which can then also be get/set by other plugins that depend on the plugin (if that ever ends up being a thing!).

(I hope this made sense).

Reply:
I think we'll get to that. Being able to "add shared state" that's namespaced for your tool or suite of tools sounds like a good value add.